repo/meta/schema/version-schema.json

```diff
@@ -1,12 +0,0 @@
-{
-  "$schema": "http://json-schema.org/schema#",
-  "type": "object",
-  "properties": {
-    "version": {
-      "type": "string",
-      "pattern": "^[0-9]\\.[0-9]\\.[0-9](-[0-9A-Za-z]+)?$"
-    }
-  },
-  "required": ["version"],
-  "additionalProperties": false
-}
```

Comment:
Legacy schema. Remove. The new Universe repo is not version and instead supports all version of packages (v2, v3 and v4) and the consumption api (v3, v4).